adamsjr8576/rancid-tomatillos
# Home styling

#### What does this PR do?
- This PR adds some styling to the Header, MoviesContainer, MovieCard components. 
- formats cards in container using flexbox
- adds inline styling to header and cards to dynamically add the backgroundImage
- creates functionality to randomly add a movie poster to background header on page load
- implements functionality to dynamically add different tomatillos based on the average rating
 - 0-3: brown tomatillo
 - 4-6: yellow tomatillo
 - 7-10: green tomatillo

#### How to test?
- pull down the changes and run `npm start` - refresh to make sure header image changes and color tomatillos match averageRating

#### Notes
- I used inline styling so that backgroundImage has access to the images within the component

#### Screenshots:

<img width="1440" alt="Screen Shot 2019-12-18 at 9 08 15 AM" src="https://user-images.githubusercontent.com/49846853/71102738-35c23900-2176-11ea-992c-8e1b925cb293.png">

@ckoga 
@JamesRexMiller4
